src/main/java/com/salesforce/dataloader/controller/Controller.java

```diff
@@ -108,6 +109,8 @@
 
     private static AppUtil.OSType OS_TYPE;
     private static boolean reuseClientConnection = true;
+    
```

Comment:
nit : can remove empty line

Reply:
I had the empty line for readability. To show that the newly added class-level field is for a distinct use-case than the previous ones.